app/admin/views.py

```diff
@@ -1,18 +1,17 @@
-from ..decorators import admin_required
-
-from flask import render_template, abort, redirect, flash, url_for
-from flask.ext.login import login_required, current_user
+from flask import abort, flash, redirect, render_template, url_for
+from flask.ext.login import current_user, login_required
 
+from . import admin
+from .. import db
+from ..decorators import admin_required
+from ..email import send_email
+from ..models import Role, User
 from forms import (
     ChangeUserEmailForm,
     NewUserForm,
     ChangeAccountTypeForm,
     InviteUserForm,
```

Comment:
Can this be alphabetical order? (also I think I actually fix this in my PR so hopefully that's okay)